apps/examples/testcase/le_tc/kernel/tc_tash_heapinfo.c

```diff
@@ -111,16 +113,21 @@ static void *heapinfo_free(void *mem, int info[4])
 }
 static void heapinfo_calc_stack(pid_t pid, int info[4], int type)
 {
-	struct tcb_s *tcb;
-	tcb = sched_gettcb(pid);
-
+	int fd;
+	int size;
+	fd = tc_get_drvfd();
+	size = ioctl(fd, TESTIOC_GET_TCB_ADJ_STACK_SIZE, pid);
+	if (size < 0) {
+		printf("heapinfo_calc_stack FAIL : ioctl(TESTIOC_GET_TCB_ADJ_STACK_SIZE)\n");
+		return NULL;
```

Comment:
1. this function return type is void. please check this.
2. if this function returns fail, original tc should be fail also.

Reply:
i fixed.